integration/gce.sh

```diff
@@ -87,72 +87,72 @@ service docker restart
 EOF
 	# It seems we need a short delay for docker to start up, so I put this in
 	# a separate ssh connection.  This installs nsenter.
-	ssh -t $name sudo docker run --rm -v /usr/local/bin:/target jpetazzo/nsenter
+	ssh -t "$name" sudo docker run --rm -v /usr/local/bin:/target jpetazzo/nsenter
 }
 
 function copy_hosts {
 	hostname=$1
 	hosts=$2
-	cat $hosts | ssh -t "$hostname" "sudo -- sh -c \"cat >>/etc/hosts\""
+	ssh -t "$hostname" "sudo -- sh -c \"cat >>/etc/hosts\"" < "$hosts"
 }
 
 # Create new set of VMs
 function setup {
 	destroy
 
-	names="$(vm_names)"
-	gcloud compute instances create $names --image $TEMPLATE_NAME --zone $ZONE
-	gcloud compute config-ssh --ssh-key-file $SSH_KEY_FILE
+	names=( $(vm_names) )
```

Comment:
given every use of names is of the form `"${names[@]}"`, why bother making a list to begin with?